app/controllers/active_admin/resource_controller.rb

```diff
@@ -15,13 +7,13 @@ class ResourceController < BaseController
     respond_to :html, :xml, :json
     respond_to :csv, only: :index
 
-    include ActionBuilder
-    include Decorators
-    include DataAccess
-    include Scoping
-    include Streaming
-    include Sidebars
-    extend  ResourceClassMethods
+    include ResourceController::ActionBuilder
+    include ResourceController::Decorators
+    include ResourceController::DataAccess
+    include ResourceController::Scoping
+    include ResourceController::Streaming
+    include ResourceController::Sidebars
+    extend  ResourceController::ResourceClassMethods
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.